index.html

```diff
@@ -1,2 +1,7 @@
 <title>C'est Bones!</title>
-<img src="tim.jpg">
+
+<script type="module">
```

Comment:
What is our browser support? I found it hard to make implementation decisions when I don't have this data in hand.

Reply:
Currently, we're supporting all browsers. Including IE6.

Reply:
Tim needs to be seen on as many platforms as possible.

Also last time I checked, Tim uses Android. So who knows what version he's on. Could be 2009.